### ISSUE TITLE
[Feature Request | ui5-rating-indicator]: a11y - switch role when readOnly and enabled to prevent misleading screenreader announcement

### ISSUE THREAD

**Issue by I585157** [SAP/ui5-webcomponents#7631]
### Feature Request Description

a11y - JAWS announcement is misleading for properties readonly is true and disabled is false. The rating indicator is missing an announcement that matches the behavior of the indicator when readOnly is true and disable is false. 

Actual behavior: when user tabs to the rating indicator the announcement ends with "to increase or decrease use the arrow keys" even though the ratingIndicator is readOnly. 

### Proposed Solution

The misleading announcement is currently from the aria role="slider". When the properties readOnly=true and disabled=false are selected, a different role could be used. 

### Organization

SuccessFactors

### Priority

Medium

### Privacy Policy

- [X] I’m not disclosing any internal or sensitive information.

**Comment by elenastoyanovaa** [comment]
Hello,

This is not a practise in our projects and we do not have such handling in any other control in UI5 or component in ui5 web components. What is more, this would mean that when a user navigates to the element he will not know the element role if removed, also when he reads the page using SR reading mode. This IMO is an accessibility issue.
Even more - the speech output is a default output coming from the screen reader itself and it has behaved like this in every web application in the web. Even though the speech output is misleading in a sense, the element is also read out as readonly and the user actually knows that he cannot interact with it. 
Also I did not find such recommendations in the aria best practise https://www.w3.org/TR/wai-aria/#aria-readonly 

Kind Regards,
Elena
